<ui version="4.0" >
 <class>qtCMBTINStitcherDialog</class>
 <widget class="QDialog" name="qtCMBTINStitcherDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TIN Stitcher Parameters</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>TIN Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="UserSpecifiedTINType" >
     <item>
      <property name="text" >
       <string>Auto Detect</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Type I</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QCheckBox" name="UseQuads" >
     <property name="text" >
      <string>Use Quads (Type I)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QCheckBox" name="AllowInteriorPointInsertion" >
     <property name="text" >
      <string>Allow Interior Point Insertion</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Tolerance:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QDoubleSpinBox" name="Tolerance" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="decimals" >
      <number>9</number>
     </property>
     <property name="maximum" >
      <double>0.050000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.000001000000000</double>
     </property>
     <property name="value" >
      <double>0.000001000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Minimum Angle:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QDoubleSpinBox" name="MinimumAngle" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="maximum" >
      <double>34.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>qtCMBTINStitcherDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>qtCMBTINStitcherDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
